recipes/econci/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "econci" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7d8aba23d45b73c64d013a03d7e6ef0fdd958d5293d6b1ff3202daaa2986a3aa"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - networkx >=2.4
```

Comment:
You only need `python` and `pip` in the `host` section. Everything else is just a run time dependency

Reply:
Done